src/templates/article.js

```diff
@@ -53,101 +220,374 @@ const PreviewContainer = styled.div`
+    width: 10px;
+`
+const RightSocialComponents = styled.div`
+    width: auto;
 `
 
-const ArticleTemplate = (props) => {
-    const post_data = props.data.directus.articles[0]
-    const main_image = post_data?.main_image?.imageFile
-    const main_video = post_data?.main_video?.imageFile?.publicURL
+const convertDate = (date) => {
+    const newdate = new Date(date)
+    return (
+        newdate.toLocaleString('en', { day: 'numeric' }) +
+        ' ' +
+        newdate.toLocaleString('en', { month: 'long' }) +
+        ' ' +
+        newdate.toLocaleString('en', { year: 'numeric' })
+    )
+}
+
+const ArticlesTemplate = (props) => {
+    const pathname = props.pageContext.pathname
+    const post_data = props.data.directus.blog[0]
+    const footer_banner_data = post_data?.footer_banners
+    const side_banner_data = post_data?.side_banners
+
+    const side_banner_data_details = {
+        max_w_value: '328px',
+        max_w_tablet: '320px',
+        isExternal: true,
+        redirectLink: side_banner_data?.cta_url,
+        imgSrcDesktop: side_banner_data?.banner_image?.imageFile,
+    }
+
+    const footer_banner_details = {
+        max_w_value: '792px',
+        max_w_tablet: '580px',
+        isExternal: true,
+        redirectLink: footer_banner_data?.cta_url,
+        imgSrcDesktop: footer_banner_data?.desktop_banner_image?.imageFile,
+        imgSrcMobile: footer_banner_data?.mobile_banner_image?.imageFile,
+    }
+
     return (
         <Layout>
             <SEO
-                description={localize('Single Article Post | Deriv')}
-                title={post_data.article_title}
+                description={
+                    'Learn about the markets that you can trade online with Deriv, including forex, commodities, synthetic indices, and stock indices.'
+                }
+                title={'Markets | Markets to trade | Deriv'}
                 no_index
             />
-            {main_image && <QueryImage data={main_image} />}
-            {main_video && (
-                <Flex>
-                    <video src={main_video} width="90%" controls />
-                </Flex>
-            )}
-            <SectionContainer>
-                <SmallContainer fd="column">
-                    <Header as="h1" type="page-title">
-                        {post_data.article_title}
-                    </Header>
-
-                    <PreviewContainer
-                        dangerouslySetInnerHTML={{ __html: post_data.article_body }}
-                    ></PreviewContainer>
-                </SmallContainer>
+
+            <SectionContainer padding="0" position="relative">
+                <Background>
+                    <HeroContainer>
+                        <HeroLeftWrapper width="100%">
+                            <InfoText mb="16px" size="14px">
+                                {post_data?.published_date &&
+                                    localize(convertDate(post_data?.published_date))}
```

Comment:
I don't think we need to localize dates.

Reply:
amended in latest commit